### DISCUSSION TITLE
Battery Drain with FreeTube

### DISCUSSION THREAD

**Discussion by Rhettica** [FreeTubeApp/FreeTube#3248] [General]
Hi,

Over the past few weeks, CleanMyMac X has reported that FreeTube is draining my battery rapidly and asking if I want to stop the program. I have noticed that the battery goes down faster than it used to, especially if I have FreeTube open for several hours. 
I was wondering if anyone else has noticed battery drain when running the program?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi there,

Does it say why it is draining fast? 

FreeTube doesn't have any background activity, it is possible that because it's an electron app, it's draining more energy than native apps.

**Reply by Rhettica** [reply]
Hi,
No, it doesn't give any information and unfortunately I can't seem to find any reason for it either. Could possibly be an issue with CleanMyMac but wanted to check if anyone else is having the problem. I'll keep an eye on it for now i guess.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Hmm, do you maybe have other Electron based applications installed. If so, are u also getting notified about those applications?

**Reply by Rhettica** [reply]
I got a notification for Brave Browser the other day but i'm not sure about other electron apps

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Looks like Brave is not using Electron.

> When I first joined Brave in 2018, our browser was still based on Muon (our Electron fork) but switched to Chromium shortly after. 

Retrieved from https://brave.com/building-brave/

We hope this issue maybe gets partially resolved by https://github.com/FreeTubeApp/FreeTube/pull/3286

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Hi @Rhettica #3286 is merged. Is it possible for you to test [the latest nightly build](https://docs.freetubeapp.io/development/nightly-builds) and report back if the issue is resolved for u.

**Reply by Rhettica** [reply]
Hi, after testing the nightly build for a while, the notification has disappeared. My battery seems to be lasting longer as well :)
